introduction_to_amazon_algorithms/imageclassification_caltech/Image-classification-incremental-training-highlevel.ipynb

```diff
@@ -10,9 +10,15 @@
     "  1. [Permissions and environment variables](#Permissions-and-environment-variables)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        epochs=2,
</pre><p>Should we add this explanation somewhere in this notebook as well?</p><p> </p><pre class="ql-syntax" spellcheck="false">After setting training parameters, we kick off training, and poll for status until training is completed, which in this example, takes between 10 to 12 minutes per epoch on a p2.xlarge machine. The network typically converges after 10 epochs. However, to save the training time, we set the epochs to 2 but please keep in mind that it may not be sufficient to generate a good model.  
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yup let me add that.